index.rst

```diff
@@ -50,10 +50,8 @@ It does not replace the required safety training to be on the summit at night.
 This includes:
 
 - procedures on who (and how) to call when there is an issue
-- location of keys, passwords etc
 - what to do in case of vehicle failure, power-outage, fire etc.
```

Comment:
What

Reply:
That line was deleted because it didn't make a lot of sense. I think it was referencing the locations of the keys to the server room, or maybe passwords for machines. We will need a separate procedure for server room, and passwords are not posted here, so this is not needed.